2006-05-28  Aurelien Jarno <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/sparc/fork.S: remove, so that fork.c is 
	  used instead.


---
 sysdeps/unix/sysv/linux/sparc/fork.S |   30 ------------------------------
 1 file changed, 30 deletions(-)

--- a/sysdeps/unix/sysv/linux/sparc/fork.S
+++ /dev/null
@@ -1,30 +0,0 @@
-/* Copyright (C) 1997, 1999, 2002 Free Software Foundation, Inc.
-   This file is part of the GNU C Library.
-   Contributed by Miguel de Icaza <miguel@gnu.ai.mit.edu>, 1997.
-
-   The GNU C Library is free software; you can redistribute it and/or
-   modify it under the terms of the GNU Lesser General Public
-   License as published by the Free Software Foundation; either
-   version 2.1 of the License, or (at your option) any later version.
-
-   The GNU C Library is distributed in the hope that it will be useful,
-   but WITHOUT ANY WARRANTY; without even the implied warranty of
-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-   Lesser General Public License for more details.
-
-   You should have received a copy of the GNU Lesser General Public
-   License along with the GNU C Library; if not, write to the Free
-   Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
-   02111-1307 USA.  */
-
-#include <sysdep.h>
-
-PSEUDO (__libc_fork, fork, 0)
-	sub	%o1, 1, %o1
-	retl
-	 and	%o0, %o1, %o0
-PSEUDO_END (__libc_fork)
-
-weak_alias (__libc_fork, __fork)
-libc_hidden_def (__fork)
-weak_alias (__libc_fork, fork)
